docs/contributing/debugging.md

```diff
@@ -4,31 +4,32 @@ title: Debugging
 
 ## Log statements
 
-TODO these are both out of date
+Both the frontend and backend have simple log statements used during local
+development, which return `()`.
 
-Both the frontend and backend have simple log statements, which return `()`.
+### Client
 
-- Frontend (prints in the browser console): `Js.log2 "my message" anyValue`
-- Backend (prints in `rundir/logs/server.log` in purple; search for
-  **"INSPECT"**): `Log.inspecT "my message" any_value`
+In the `client`, you may log with either of:
 
-If you have existing code like this:
+- `Js.log2("my message", anyValue)`
+- `Debug.loG("my message", anyValue)` Both of these print in the browser
```

Comment:
I see both of these in use but I've only used the latter - not sure why.
Is there any intention to migrate Js.log2 usages to fns in the `Debug` module? Or when should someone use one over the over?

Reply:
Debug.log is more flexible, no plans but no reasons to stick with log2 afaik